### PR TITLE
[15.0][MIG] account_asset_management_menu

### PR DESCRIPTION
**Pull request by JuanyDForgeflow**
Users now need to make sure to activate the setting Show full accounting features in order to display the asset menus.

### CODE REVIEW

**Comment by JuanyDForgeflow** [comment]
This module depends on https://github.com/OCA/account-financial-tools/pull/1275

File: account_asset_management_menu/__manifest__.py [hunk 1]
```diff
@@ -0,0 +1,14 @@
+# Copyright 2021 ForgeFlow, S.L.
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl).
+
+{
+    "name": "Assets Management Menu",
+    "version": "15.0.1.0.0",
+    "license": "AGPL-3",
+    "depends": ["account_asset_management", "account_menu"],
```
Comment (mariadforgeflow) [inline account_asset_management_menu/__manifest__.py]:
```suggestion
    "depends": ["account_asset_management", "account_usability"],
```
As you can see in https://github.com/OCA/account-financial-tools/pull/1275/, account_menu has been renamed to account_usability

Comment (JuanyDForgeflow) [reply account_asset_management_menu/__manifest__.py]: @mariadforgeflow I added the dependency

**Review by mariadforgeflow — APPROVED** [review]

**Review by AaronHForgeFlow — APPROVED** [review]
LGTM

**Pull request merged**